ravenframework/utils/InputData.py

```diff
@@ -661,14 +664,15 @@ def generateParamsLatex(cls, recDepth):
       typ = info['type'].generateLatexType()
       req = 'required' if info['required'] else 'optional'
       default = '\\default{'+ str(info['default']) +'}' if info['default'] != 'no-default' else ""
+      default2 = 'default='+ str(info['default']) if info['default'] != 'no-default' else ""
       desc = wrapText(info['description'], indent=doDent(recDepth, 3))
-      if default:
+      if default or default2:
         msg += '\n{i}  \\item \\xmlAttr{{{n}}}: \\xmlDesc{{{t}, {r}}}, \n{d} {de}'.format(i=doDent(recDepth, 1),
                                                                                      n=name,
                                                                                      t=typ,
                                                                                      r=req,
                                                                                      d=desc,
-                                                                                     de=default)
+                                                                                     de=default2)
```

Comment:
why default and default2 both used here? It seems to me you only need 1.

Reply:
Some other places might still have the \default in the description. Should I just leave default2?

Reply:
I may not understand the logic here, it seems to me 'default' and 'default2' are similar.